pypeit/scripts/tellfit.py

```diff
@@ -166,6 +166,7 @@ def main(args):
                                            tell_norm_thresh=par['telluric']['tell_norm_thresh'],
                                            only_orders=par['telluric']['only_orders'],
                                            bal_wv_min_max=par['telluric']['bal_wv_min_max'],
+                                           resln_frac_bounds=par['telluric']['resln_frac_bounds'],
```

Comment:
Is this defined by `TelluricPar` in `par/pypeitpar.py`?

Reply:
Yep, it's already there.